### PR TITLE
Button refacto

### PR DESCRIPTION
**Pull request by AlexandreBroudin**
Back to original paddings, you can overide them by using `$btn-base-pd-vertical` & `$btn-base-pd-horizontal` if you need smaller or bigger buttons

### CODE REVIEW

File: src/scss/components/button/_base.scss [hunk 1]
```diff
@@ -1,6 +1,9 @@
+$btn-base-pd-vertical: $pd--s;
```
Comment (ugogo) [inline src/scss/components/button/_base.scss:1]: Variables should in `_core.scss` no?
cc @GabrielCousin

**Comment by AlexandreBroudin** [comment]
I don't know.
_core.scss ? or _var.scss ?
We don't have a real solution

File: src/scss/components/button/button.md [hunk 2]
```diff
@@ -26,28 +26,33 @@
...
 ## Types
 
+### Raw
 ```html
 <button class="Btn Btn--raw">Raw button</button>
 ```
 
+### Rounded
+```html
+<button class="Btn Btn--rounded">
```
Comment (GabrielCousin) [inline src/scss/components/button/button.md:40]: `Btn--circle`, rounded is more confusing.

File: src/scss/components/button/_base.scss (hunk 1 above)
Comment (GabrielCousin) [inline src/scss/components/button/_base.scss:1]: Approved!

File: src/scss/components/button/button.md (hunk 2 above)
Comment (AlexandreBroudin) [inline src/scss/components/button/button.md:40]: You're right
rounded https://www.google.fr/search?q=button+circle&source=lnms&tbm=isch&sa=X&ei=3r-NVaTzOsGsU5GEi7gC&ved=0CAcQ_AUoAQ&biw=1276&bih=714#tbm=isch&q=button+rounded

vs 

circle https://www.google.fr/search?q=button+circle&source=lnms&tbm=isch&sa=X&ei=3r-NVaTzOsGsU5GEi7gC&ved=0CAcQ_AUoAQ&biw=1276&bih=714#tbm=isch&q=button+rounded

File: src/scss/components/button/_base.scss [hunk 3]
```diff
@@ -1,6 +1,9 @@
...
+$btn-base-pd-horizontal: $pd--m - ($fz--m/2);
```
Comment (GabrielCousin) [inline src/scss/components/button/_base.scss:2]: I don't agree with this. It is squeezing all buttons, just in order to fix the circle one.
Why not adding a specific horizontal padding only on `.Btn--circle`?

**Pull request closed without merging**